README.md

```
@@ -122,6 +122,38 @@ type CustomIndexes = {
 }
 ```
 
+<details>
```

markdownlint-cli2 (0.17.2)

Inline HTML
Element: details

(MD033, no-inline-html)